src/components/General/Search.vue

```diff
@@ -13,6 +13,7 @@
         @selected="selected"
       />
     </q-search>
+    <span v-if="!results.length" >No results.</span>
```

Comment:
All the text needs to be internationalized so we can manage the translations.

Have a look at the other files for examples of how to do this, e.g. from CurrentUser.vue there is:

```vue
<span>{{ $t('PICKUPLIST.ITEM.LEAVE') }}</span>
```

Reply:
Will try